spec/models/event_spec.rb

```diff
@@ -17,4 +17,25 @@
       expect(event).to be_valid
     end
   end
+
+  describe '#in_week' do
+    let(:start_time) { Time.now.beginning_of_week }
+    let!(:events) {
+      [
+          FactoryGirl.create(:event, start_time: start_time, end_time: start_time + 2.hours),
+          FactoryGirl.create(:event, start_time: start_time + 1.days, end_time: start_time + 1.days + 2.hours),
+          FactoryGirl.create(:event, start_time: start_time - 2.days, end_time: start_time - 2.days + 1.hours),
+          FactoryGirl.create(:event, start_time: start_time - 2.days, end_time: start_time + 2.days),
+          FactoryGirl.create(:event, start_time: start_time - 2.days, end_time: start_time + 10.days),
+          FactoryGirl.create(:event, start_time: start_time + 1.days, end_time: start_time + 10.days)
+
+      ]
+    }
+
+    it "returns all events from specified week" do
+      expected_events = events.clone.tap { |n| n.delete_at(2) }
```

Comment:
don't do clone and then sort the collection just to compare them, here's a few matchers for collections:
`expect(actual).to include(expected)
expect(array).to match_array(expected_array)
# ...which is the same as:
expect(array).to contain_exactly(individual, elements)`